File: 3.js
// Функция для реверса цифр числа
function reverseNumber(num) {
    let reversed = 0;
    for (let n = num; n > 0; n = Math.floor(n / 10)) {
        reversed = reversed * 10 + n % 10;
    }
    return reversed;
}

// Основная функция с красивым выводом
function calculateDifferences(...numbers) {
    console.log("Результаты:");
    numbers.forEach(num => {
        const reversed = reverseNumber(num);
        const diff = Math.abs(num - reversed);
        console.log(|${num} - ${reversed}| = ${diff});
    });
}

// Пример использования
calculateDifferences(123, 400);

File: 3.kt
fun reverseNumber(num: Int): Int {
    var reversed = 0
    var n = num
    while (n > 0) {
        reversed = reversed * 10 + n % 10
        n /= 10
    }
    return reversed
}
fun main() {
    print("Введите количество чисел: ")
    val n = readln().toInt()
    
    print("Введите числа через пробел: ")
    val numbers = readln()
        .split(" ")
        .map { it.toInt() }
        .take(n)
    
    println("Результаты:")
    numbers.forEach { number ->
        val reversed = reverseNumber(number)
        val difference = kotlin.math.abs(number - reversed)
        println("|$number - $reversed| = $difference")
    }
}
